package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bfx-api-node-rest",
-  "version": "3.0.9",
+  "version": "3.1.0",
```

Comment:
why not 3.0.10 ?

Reply:
^ agreed, only minor additions here so we can use 3.0.10

Reply:
Sure, sending.